docroot/modules/humsci/hs_layouts/hs_layouts.install

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/**
+ * @file
+ * Update functions for hs_layouts module.
+ */
+
+use Drupal\layout_builder\Section;
+
+/**
+ * Set main content id in content types default layout builder config.
+ */
+function hs_layouts_update_9401() {
+  $storage = \Drupal::entityTypeManager()->getStorage('entity_view_display');
+  $query = $storage->getQuery()
+    ->condition('targetEntityType', 'node')
+    ->condition('mode', 'default');
+  $items = $query->execute();
+  foreach ($items as $item) {
+    $view_display = $storage->load($item);
+    if ($view_display->isLayoutBuilderEnabled()) {
+      $sections = $view_display->getThirdPartySetting('layout_builder', 'sections');
```

Comment:
`$view-display->getSections()` would be more concise here.

Reply:
Fixed.